module_utils/foreman_helper.py

```diff
@@ -244,6 +244,9 @@ def _generate_resource_params(self, resource, action, resource_id=None, params=N
                 resource_params[hash_entry_name] = params
             else:
                 resource_params.update(params)
+        if action == 'create':
+            if resource == 'organizations':
```

Comment:
both if statements could be written in one with an and

Reply:
I thought since we probably will need the same for location it is better to have a decision tree.

Reply:
looking at apidocs, locations does not have this oddity

Reply:
fair enough